src/main/app/src/assets/i18n/en.json

```diff
@@ -72,6 +72,8 @@
   " 3 ": ">>--> OVERIGE VELDEN <--<<",
   "initiator": "Applicant",
   "auteur": "Author",
+  "ondertekenen": "Sign document",
```

Comment:
Ondertekenen is een actie toch? Zet die dan ook in de vertalingen bij de acties.